migrations/20210510031123-add-garden-plant-references.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
```

Comment:
This file was created using the following command run inside the server container:
```
sequelize migration:generate --name add-garden-plant-references
```
This command creates an EMPTY migration, which I then filled with the code to add FOREIGN KEY and UNIQUE constraints into the database for the MANY-TO-MANY join table.